lib/spack/spack/cmd/external.py

```diff
@@ -91,6 +105,32 @@ def setup_parser(subparser):
         help=("if a manifest file cannot be parsed, fail and report the " "full stack trace"),
     )
 
+    edit_parser = sp.add_parser("edit", help="open packages.yaml in $EDITOR")
```

Comment:
I think it makes sense to drop the `edit` command since, afaict, it is already covered by `spack config edit packages` as @alalazo pointed out.